cardano-api/src/Cardano/Api/LedgerEvent.hs

```diff
@@ -153,23 +155,33 @@ pattern LEMirTransfer rp tp rtt ttr <-
         ( MirEvent
             ( MirTransfer
                 ( InstantaneousRewards
-                    (convertSumRewardsMap -> rp)
-                    (convertSumRewardsMap -> tp)
+                    (convertSumRewardsMapSimple -> rp)
+                    (convertSumRewardsMapSimple -> tp)
                     (fromShelleyDeltaLovelace -> rtt)
                     (fromShelleyDeltaLovelace -> ttr)
                   )
               )
           )
       )
 
-convertSumRewardsMap ::
+convertRewardEventMapNew ::
```

Comment:
Can we drop the `New` here? The names don't clash

Reply:
There are two functions named `convertSumRewardsMap*`. How would you differentiate them?